python/pfs/drp/stella/synthetic.py

```diff
@@ -290,12 +291,13 @@ def makeSyntheticPfsConfig(config, pfsDesignId, visit, rng=None,
 
     fiberStatus = np.full_like(targetType, FiberStatus.GOOD)
 
-    fiberMag = [np.array([22.0, 23.5, 25.0, 26.0] if
-                         tt in (TargetType.SCIENCE, TargetType.FLUXSTD) else [])
-                for tt in targetType]
+    fbr_flx = [(m*u.ABmag).to_value(u.nJy) for m in [22.0, 23.5, 25.0, 26.0]]
```

Comment:
Fix `fbr_flx`.

Reply:
This calculation computes the flux in [nJy] from the AB magnitude. Can you elaborate a little more as to the error?